Arm/Hand/wrist_joint.hpp

```diff
@@ -32,46 +39,40 @@ class WristJoint
     right_motor_.SetAngle(angle_to_degrees);
   }
 
-  void SetRollPosition(float roll_angle)
-  {
-    roll_angle_ = roll_angle;
-    units::angle::degree_t angle_to_degrees(roll_angle);
-    left_motor_.SetAngle(angle_to_degrees);
-    right_motor_.SetAngle(angle_to_degrees);
-  }
-
-  /// Sets the offset values to find its true zeroing point
-  void SetZeroOffsets(float left_offset, float right_offset)
+  // Move the wrist to its callibrated roll and pitch angles
+  void SetPosition(float pitch_angle,
+                   float roll_angle)
   {
-    left_offset_angle_  = left_offset;
-    right_offset_angle_ = right_offset;
+    //units::angle::degree_t pitch_angle_to_degree(pitch_angle);
+    //units::angle::degree_t roll_angle_to_degree(roll_angle);
   }
 
-  /// Returns the acceleration values from the MPU6050
-  sjsu::Accelerometer::Acceleration_t GetAccelerometerData()
+  /// Sets the zero_offset_angle value that the motors use to know its true '0'
+  /// position. Called by RoverArmSystem::Home
+  void SetZeroOffsets(float left_offset,
+                      float right_offset)
   {
-    return mpu_.Read();
+    units::angle::degree_t left_offset_to_degree(left_offset);
+    units::angle::degree_t right_offset_to_degree(right_offset);
+    left_zero_offset_angle  = left_offset_to_degree;
+    right_zero_offset_angle = right_offset_to_degree;
   }
 
-  int GetPitchPosition()
+  /// Return the acceleration values for the MPU6050 on the joint.
+  Acceleration GetAccelerometerData()
   {
-    return int(pitch_angle_);
+    sjsu::Accelerometer::Acceleration_t acceleration_to_float(mpu.Read());
+    Acceleration acceleration;
+    acceleration.x = static_cast<float>(acceleration_to_float.x);
+    acceleration.y = static_cast<float>(acceleration_to_float.y);
+    acceleration.z = static_cast<float>(acceleration_to_float.z);
+    return acceleration;
   }
 
-  int GetRollPosition()
-  {
-    return int(roll_angle_);
-  }
-
- private:
-  sjsu::RmdX & left_motor_;
-  sjsu::RmdX & right_motor_;
-  sjsu::Mpu6050 & mpu_;
-
-  float pitch_angle_        = 0;
-  float roll_angle_         = 0;
-  float left_offset_angle_  = 0;
-  float right_offset_angle_ = 0;
+int GetRollPosition(){
+  return 0;
+}
```

Comment:
Add the deleted logic back, this was a tricky merge conflict. That's why it's good to merge in PR asap